tools/mafft/macros.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <macros>
     <token name="@VERSION@">0</token>
-    <token name="@TOOL_VERSION@">7.508</token>
+    <token name="@TOOL_VERSION@">7.52</token>
```

Comment:
Oh wow, is 7.52 really bigger/newer than 7.508?

Reply:
The latest version is actually `7.520`. @elischberg I don't think you can just drop that trailing zero.

Reply:
Oh my bad. I thought, the dot is a comma.